modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -35,12 +35,17 @@ PERF_TEST_P_(AddPerfTest, TestPerformance)
     out = cv::gapi::add(in1, in2, dtype);
     cv::GComputation c(GIn(in1, in2), GOut(out));
 
+    // There is no need to qualify gin, gout, descr_of with namespace (cv::)
+    // as they are in the same namespace as their actual argument (i.e. cv::Mat)
+    // and thus are found via ADL, as in the examples below.
     // Warm-up graph engine:
-    c.apply(gin(in_mat1, in_mat2), gout(out_mat_gapi), std::move(compile_args));
+    auto cc = c.compile(descr_of(gin(in_mat1, in_mat2)),
+                        std::move(compile_args));
+    cc(gin(in_mat1, in_mat2), gout(out_mat_gapi));
 
     TEST_CYCLE()
     {
-        c.apply(gin(in_mat1, in_mat2), gout(out_mat_gapi), std::move(compile_args));
+        cc(in_mat1, in_mat2, out_mat_gapi);
```

Comment:
let's use the same (i.e. main one) overload of `operator()` in performance test cycles as well. 
I.e. one that utilized easily with use of `gin()` and `gout()`. 

Please update rest of the path accordingly

Reply:
Updated